lib/params/GUIStateParams.cpp

```diff
@@ -46,6 +46,8 @@ const string GUIStateParams::_flowDimensionalityTag = "_flowDimensionalityTag";
 const string GUIStateParams::BookmarksTag = "BookmarksTag";
 const string GUIStateParams::MovingDomainTrackCameraTag = "MovingDomainTrackCameraTag";
 const string GUIStateParams::MovingDomainTrackRenderRegionsTag = "MovingDomainTrackRenderRegionsTag";
+const string GUIStateParams::SelectedImportDataTypeTag = "SelectedImportDataTypeTag";
+const string GUIStateParams::SelectedImportDataFilesTag = "SelectedImportDataFilesTag";
```

Comment:
Do these need to be persistent state?

Reply:
I think it's convenient for the user.  If they import a data type once, they'll probably want to import the same type again if needed.